test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -91,24 +91,37 @@ public class CcmBridge implements AutoCloseable {
   public static final File DEFAULT_CLIENT_PRIVATE_KEY_FILE = createTempStore("/client.key");
   public static final File DEFAULT_CLIENT_CERT_CHAIN_FILE = createTempStore("/client.crt");
 
-  public static final String DEFAULT_SERVER_TRUSTSTORE_PASSWORD = "fakePasswordForTests";
+  public static final String DEFAULT_SERVER_TRUSTSTORE_PASSWORD = "scylla1sfun";
   public static final String DEFAULT_SERVER_TRUSTSTORE_PATH = "/server.truststore";
 
+  public static final String DEFAULT_SERVER_TRUSTSTORE_PEM_PATH = "/server.truststore.pem";
+
   private static final File DEFAULT_SERVER_TRUSTSTORE_FILE =
       createTempStore(DEFAULT_SERVER_TRUSTSTORE_PATH);
+  private static final File DEFAULT_SERVER_TRUSTSTORE_PEM_FILE =
+      createTempStore(DEFAULT_SERVER_TRUSTSTORE_PEM_PATH);
 
-  public static final String DEFAULT_SERVER_KEYSTORE_PASSWORD = "fakePasswordForTests";
+  public static final String DEFAULT_SERVER_KEYSTORE_PASSWORD = "scylla1sfun";
   public static final String DEFAULT_SERVER_KEYSTORE_PATH = "/server.keystore";
 
+  // Contain the same keypair as the server keystore, but in format usable by Scylla
+  public static final String DEFAULT_SERVER_PRIVATE_KEY_PATH = "/server.key";
+  public static final String DEFAULT_SERVER_CERT_CHAIN_PATH = "/server.crt";
+
   private static final File DEFAULT_SERVER_KEYSTORE_FILE =
       createTempStore(DEFAULT_SERVER_KEYSTORE_PATH);
+  private static final File DEFAULT_SERVER_PRIVATE_KEY_FILE =
+      createTempStore(DEFAULT_SERVER_PRIVATE_KEY_PATH);
+  private static final File DEFAULT_SERVER_CERT_CHAIN_FILE =
+      createTempStore(DEFAULT_SERVER_CERT_CHAIN_PATH);
 
   // A separate keystore where the certificate has a CN of localhost, used for hostname
   // validation testing.
   public static final String DEFAULT_SERVER_LOCALHOST_KEYSTORE_PATH = "/server_localhost.keystore";
 
-  private static final File DEFAULT_SERVER_LOCALHOST_KEYSTORE_FILE =
-      createTempStore(DEFAULT_SERVER_LOCALHOST_KEYSTORE_PATH);
+  private static final File DEFAULT_SERVER_LOCALHOST_KEYSTORE_FILE = null;
+  // @IntegrationTestDisabledCassandra3Failure @IntegrationTestDisabledSSL
+  // = createTempStore(DEFAULT_SERVER_LOCALHOST_KEYSTORE_PATH);
```

Comment:
What is this comment for?

Reply:
It is for a time when we fix that remaining test as a reminder of the original (correct) implementation.